runtime/src/main/java/com/datastax/oss/quarkus/runtime/api/config/CassandraClientConnectionConfig.java

```diff
@@ -39,4 +39,15 @@
   /** How long the driver waits for a request to complete. */
   @ConfigItem(name = "request.timeout")
   public Optional<Duration> requestTimeout;
+
+  /** The auth provider class. If not specified, PlainTextAuthProvider will be used */
+  public Optional<String> authProviderClass;
```

Comment:
I think the auth provider class is not useful here, because the only settings exposed are `username` and `password`, which means that the only auth provider that can work is `PlainTextAuthProvider` anyways. I suggest that instead you set this option to `PlainTextAuthProvider` in `CassandraClientProducer` if `username` and `password` were provided.

Reply:
I think it defaults to PlainTextAuthProvider. Let me verify that and then I will simply drop the auth provider setting.